matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -923,6 +923,10 @@ var PhenoTips = (function (PhenoTips) {
                 this._contactContainer.down('textarea[name="message"]').update('');
                 this._contactContainer.down('input[name="subject"]').value = '';
                 this._contactContainer.down('input[name="to"]').value = '';
+                // clone send button to remove all event handlers
```

Comment:
if this works lets keep it for now, but this looks hack-ish, what event handlers do we try to remove and why?